build/cqs.tsv

```diff
@@ -200,6 +199,7 @@ org.eclipse.xsd:org.eclipse.xsd	2.12.0	compile
 org.ehcache:sizeof	0.3.0	compile
 org.ejml:ejml-core	0.34	compile
 org.ejml:ejml-ddense	0.34	compile
+org.ejml:ejml-simple	0.34	compile
```

Comment:
Other versions of ejml-simple are on ClearlyDefined: https://clearlydefined.io/?sort=releaseDate&sortDesc=true&name=ejml-simple

Thoughts?

Reply:
looks like 0.34 is on there too, declared as apache 2.0. good enough i guess? https://clearlydefined.io/definitions/maven/mavencentral/org.ejml/ejml-simple/0.34

Reply:
well actually it's also in that link you sent with 'discovered' as apache 2.0